src/rabbit_vhost.erl

```diff
@@ -261,10 +261,19 @@ infos(Items, X) -> [{Item, i(Item, X)} || Item <- Items].
 
 i(name,    VHost) -> VHost;
 i(tracing, VHost) -> rabbit_trace:enabled(VHost);
-i(state,   VHost) -> case rabbit_vhost_sup_sup:is_vhost_alive(VHost) of
-                         true -> running;
-                         false -> down
-                     end;
+i(cluster_state, VHost) ->
+    Nodes = rabbit_nodes:all_running(),
+    lists:map(fun(Node) ->
+        State = case rabbit_misc:rpc_call(Node,
+                                          rabbit_vhost_sup_sup, is_vhost_alive,
+                                          [VHost]) of
+            {badrpc, nodedown} -> nodedown;
+            true               -> running;
+            false              -> down
```

Comment:
Rather than `down` should this atom be `vhostdown` or `vhost_down` to describe it better? I saw the changes [here](https://github.com/rabbitmq/rabbitmq-management/blob/rabbitmq-management-460/priv/www/js/formatters.js#L936) and didn't know what `down` meant without coming to this line of code.